README.md

```diff
@@ -91,6 +91,15 @@ module.exports = {
   onUserLogged: (app, user) => {
       return Promise.resolve(user)
   },
+  //Optional: can be used to merge data from the oAuth / oAuth2 profile and the default user properties.
```

Comment:
It's not only oAuth and oAuth2, it's all third party login. They are mainly these 2 but they're some others